torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20762,6 +20792,22 @@ def __init__(
     #
     # Elementwise Ternary Reference OpInfos
     #
+    ElementwiseBinaryPythonRefInfo(
+        "_refs.clamp_min",
+        torch_opinfo_name="clamp_min",
+        supports_nvfuser=False,
+        skips=(
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_errors'),
```

Comment:
The same expectedFailure is under `_refs.minimum` & `_refs.maximum`, which doesn't have a comment explaining why.

Since our ref uses those as well, should be the same root cause.

Reply:
OK but what is the root cause? (Although this is probably moot because of the proposed switch to `where` above)